### PR TITLE
Adds MQTT backend

### PR DESCRIPTION
**Pull request by Governa**

### CODE REVIEW

**Comment by Alex-CodeLab** [comment]
this seems to be outdated. (it does not seem to work with the example code). 
Do you plan on fixing the merge conflict?

File: setup.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def get_packages(package):
         "redis": ["asyncio-redis"],
         "postgres": ["asyncpg"],
         "kafka": ["aiokafka"],
+        "mqtt": ["asyncio-mqtt"]
```
Comment (Kludex) [inline setup.py:52]:
```suggestion
        "mqtt": ["asyncio-mqtt"],
```

**Comment by Governa** [comment]
Most of the problems are fixed. What is still missing is a way to create a MQTT service in a GitHub workflow.

Inside the `eclipse-mosquitto` image, a file called `/mosquitto-no-auth.conf` can be used to set up a server accepting anonymous users. What we want to do is to pass `-c /mosquitto-no-auth.conf` as a parameter to the CMD of the docker image.

**Comment by lukepighetti** [comment]
This PR looks like it's real close to being merged and has some nice features

**Comment by alex-oleshkevich** [comment]
Superseded by https://github.com/encode/broadcaster/pull/117

**Pull request closed without merging**